src/pages/Index.vue

```diff
@@ -20,17 +20,18 @@
                 Building a bridge between the end-users and quantum computing
                 with visual, easy-to-use software development tools.
               </p>
-              <p>
-                We closed Quantum Flytrap in Oct 2022, see <a class="text-link" href="/blog/closing-quantum-flytrap">our post</a>.
-              </p>
             </div>
-
+            <a
+              href="https://vote.webbyawards.com/PublicVoting#/2023/websites-and-mobile-sites/general-websites-and-mobile-sites/science"
+            >
+              <g-image src="~/imgs/webby_awards_bar.png" />
+            </a>
```

Comment:
Would it be better if it opened in a new tab?

```html
<a
  href="https://vote.webbyawards.com/PublicVoting#/2023/websites-and-mobile-sites/general-websites-and-mobile-sites/science" 
  target="_blank" 
  rel="noreferrer noopener"
>
  <g-image src="~/imgs/webby_awards_bar.png" />
</a>
```

Reply:
Not sure; up to Klem